### ISSUE TITLE
Select features for PWA

### ISSUE THREAD

**Issue by vergilius** [burakukula/kazimir-pwa#4]
Service worker advanced features:
- [Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API) - might be super helpful, imagine receiving notification, when you are close to some place on map,
- [Push API](https://developer.mozilla.org/en-US/docs/Web/API/Push_API) - we are not planing to have sever so it will not be that useful,
- [Background sync API](https://developers.google.com/web/updates/2015/12/background-sync) - I don't think we need this one,
- [Channel messaging API](https://developer.mozilla.org/en-US/docs/Web/API/Channel_Messaging_API) - again, this is used with web workers, which we don't plan to use

Basically, we should take full advantage from Service worker caching features, and later try to use Notifications API.

**Comment by vergilius** [comment]
fyi @burakukula
